src/frontend/screens/Observation/ObservationView.js

```diff
@@ -86,14 +86,14 @@ const InsetMapView = React.memo<MapProps>(({ lon, lat }: MapProps) => {
       pitchEnabled={false}
       rotateEnabled={false}
       compassEnabled={false}
-      styleURL={styleUrl}
+      styleURL={selectedMap.styleUrl}
     >
       <MapboxGL.Camera
         centerCoordinate={[lon, lat]}
-        zoomLevel={styleType === "fallback" ? 5 : 12}
+        zoomLevel={selectedMap.isOfflineFallback ? 5 : 12}
         animationMode="moveTo"
       />
-      {styleType === "fallback" ? <OfflineMapLayers /> : null}
+      {selectedMap.isOfflineFallback ? <OfflineMapLayers /> : null}{" "}
```

Comment:
What is the `{" "}` for? (curiosity)

Reply:
ah think it was mistakenly added while fumbling on my keyboard 😅 . removing

Reply:
addressed via https://github.com/digidem/mapeo-mobile/pull/1048/commits/902826b8f93249dcac8bf15ef523c2d00fbb7d05